frontend/src/pages/eventoverview.js

```diff
@@ -0,0 +1,174 @@
+    this.setState({ modal: true })
+  }
+
+  closeModal = () => {
+    this.setState({ modal: false })
+  }
+
+  handleChange = e => {
+    this.setState({
+      [e.target.name]: e.target.value
+    })
+  }
+
+  handleKeyPress = e => {
+    if (e.charCode === 13) {
+      this.addEvent()
+    }
+  }
+
+  addEvent = async () => {
+    let event = {
+      name: this.state.name,
+      date: this.state.date,
+      startTime: this.state.startTime,
+      endTime: this.state.endTime,
+      location: this.state.location,
+      description: this.state.description
+    }
+    await createEvent(event)
+    this.closeModal()
+    Router.push({ pathname: '/eventoverview' })
+  }
+
+  render() {
+    return (
+      <>
+        <Nav />
+        <div className="page-content-wrapper">
+          <Container fluid>
+            <Modal isOpen={this.state.modal}>
+              <ModalHeader>Add New Event</ModalHeader>
+              <ModalBody>
+                <Form>
+                  <FormGroup>
+                    <Label>Name</Label>
+                    <Input
+                      name="name"
+                      placeholder="i.e. Product Showcase"
+                      onChange={this.handleChange}
+                    />
+                  </FormGroup>
+                  <FormGroup>
+                    <Label>Date</Label>
+                    <Input name="date" placeholder="i.e. 06/20/19" onChange={this.handleChange} />
+                  </FormGroup>
+                  <FormGroup>
+                    <Label>Start Time</Label>
+                    <Input name="startTime" placeholder="i.e. 5" onChange={this.handleChange} />
+                  </FormGroup>
+                  <FormGroup>
+                    <Label>End Time</Label>
+                    <Input name="endTime" placeholder="i.e. 6" onChange={this.handleChange} />
+                  </FormGroup>
+                  <FormGroup>
+                    <Label>Location</Label>
+                    <Input name="location" placeholder="i.e. ECEB" onChange={this.handleChange} />
+                  </FormGroup>
+                  <FormGroup>
+                    <Label>Description</Label>
+                    <Input
+                      name="description"
+                      placeholder="i.e. Teams present their final product."
+                      onChange={this.handleChange}
+                      onKeyPress={this.handleKeyPress}
+                    />
+                  </FormGroup>
+                </Form>
+              </ModalBody>
+              <ModalFooter>
+                <Button color="secondary" onClick={this.closeModal}>
+                  Cancel
+                </Button>
+                <Button color="primary" onClick={this.addEvent}>
+                  Submit
+                </Button>
+              </ModalFooter>
+            </Modal>
+            <Table>
+              <thead>
+                <ActionButton
+                  style={{ marginBottom: 15 }}
```

Comment:
Avoid inline styling if possible

Reply:
+1 in this case, the marginBottom style can be reused across multiple components as well so it's good to place this in a css/scss/etc. file

Reply:
Also inline styling is very hard to override if it's ever needed, so putting them in a css file is better for more customizable styling

Reply:
should i put css files in the static folder

Reply:
yea, I think we only have one css file and it's in there, so I'd use that.

Reply:
wait in the actionButton component it looks like i have to pass in the style as props